docs/modules/guides/pages/defining-routes.adoc

```diff
@@ -397,45 +380,37 @@ make multiple rows in a table, you will get errors:
 
 [source,clojure]
 ----
-;;; This won't work in table syntax. Both rows get the same automatic
-;;; route name.
+;;; This won't work; both rows get the same automatic
+;;; route name and an exception is thrown.
 ["/users" :get user-api-handler]
 ["/users" :post user-api-handler]
 ----
 
-You have a couple of options. To stick with table syntax, you can use
-route names to distinguish the rows:
+The best approach is to simply provide a unique route name to each route that shares
+the handler.
 
 [source,clojure]
 ----
 ["/users" :get  user-api-handler :route-name :users-view]
 ["/users" :post user-api-handler :route-name :user-create]
 ----
 
-The route names are enough to make each row unique.
-
-With terse syntax, one path has a map that allows multiple verbs. Each
-verb can use the same handler as long as they specify different route names:
-
-[source,clojure]
-----
-["/users" {:get user-api-handler
-           :post [:user-create user-api-handler]}]
-----
-
 == Generating URLs
 
-In addition to routing, route tables are also used for URL
-generation. You can request a URL for a given route by name and
+In addition to routing incoming requests, the routing data is also
+available URL generation. You can request a URL for a given route by name and
```

Comment:
available for URL

Reply:
I've had to reverse engineer the intent of the OGs, that's what I'm trying to capture in all the documentation ... but spotting little mistakes is valuable too.